src/be/personIdentifierHelpers.ts

```diff
@@ -0,0 +1,111 @@
+import { strings, isValidDateCompactYYYYMMDD } from '../util';
```

Comment:
Can we get a dedicated unit test for this file.  I do get that it's covered in other tests, but just for consistency.

Reply:
Yes, should it be for all functions in the file or just the ones that are exported?

Reply:
Just the exported ones